### PR TITLE
README Update + Progress Bar Text Clarification

### PR DESCRIPTION
**Pull request by matjin**
Minor update to README to link the 1.2 blog, as well as a minor text clarification fix for progress updates.

### CODE REVIEW

**Review by xi-liu-ds — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -5,7 +5,8 @@
 In addition to implementing the C# frontend, we contributed our [language-agnostic serialization layer](https://github.com/microsoft/infersharp/wiki/InferSharp:-A-Scalable-Code-Analytics-Tool-for-.NET#language-agnostic-representation-of-sil) ([Commit #1361](https://github.com/facebook/infer/commit/285ddb4a98f337a40d61e73b7a0867e44fa4f042)) to facebook/infer, which opens up opportunities for [additional language support](https://github.com/microsoft/infersharp/wiki/InferSharp:-A-Scalable-Code-Analytics-Tool-for-.NET#overview) in the future.
 
 ## Public Announcements
-- [.NET Blog](https://devblogs.microsoft.com/dotnet/infer-interprocedural-memory-safety-analysis-for-c/)
+- [.NET Blog 1.2](https://devblogs.microsoft.com/dotnet/infer-v1-2-interprocedural-memory-safety-analysis-for-c/)
+- [.NET Blog 1.0](https://devblogs.microsoft.com/dotnet/infer-interprocedural-memory-safety-analysis-for-c/)
```
Comment (xi-liu-ds) [inline README.md:9]: .NET Blog for InferSharp 1.0 sounds better?

File: README.md [hunk 2]
```diff
@@ -5,7 +5,8 @@
 In addition to implementing the C# frontend, we contributed our [language-agnostic serialization layer](https://github.com/microsoft/infersharp/wiki/InferSharp:-A-Scalable-Code-Analytics-Tool-for-.NET#language-agnostic-representation-of-sil) ([Commit #1361](https://github.com/facebook/infer/commit/285ddb4a98f337a40d61e73b7a0867e44fa4f042)) to facebook/infer, which opens up opportunities for [additional language support](https://github.com/microsoft/infersharp/wiki/InferSharp:-A-Scalable-Code-Analytics-Tool-for-.NET#overview) in the future.
 
 ## Public Announcements
-- [.NET Blog](https://devblogs.microsoft.com/dotnet/infer-interprocedural-memory-safety-analysis-for-c/)
+- [.NET Blog 1.2](https://devblogs.microsoft.com/dotnet/infer-v1-2-interprocedural-memory-safety-analysis-for-c/)
```
Comment (xi-liu-ds) [inline README.md:8]: .NET Blog for InferSharp 1.2 sounds better?

File: README.md (hunk 1 above)
Comment (matjin) [reply README.md]: Edited

File: README.md (hunk 2 above)
Comment (matjin) [reply README.md]: Edited

**Review by xi-liu-ds — APPROVED** [review]

**Pull request merged**